src/display/editor/draw.js

```diff
@@ -988,7 +988,7 @@ class DrawingEditor extends AnnotationEditor {
   /** @inheritdoc */
   renderAnnotationElement(annotation) {
     annotation.updateEdited({
-      rect: this.getRect(0, 0),
+      rect: this.getPDFRect(),
```

Comment:
The integration test seem to fail because there doesn't appear to be a base implementation of this. From the logs:

`JavaScript error: http://127.0.0.1:35343/build/generic/build/pdf.mjs, line 23324: TypeError: this.getPDFRect is not a function`

Reply:
Strange... I had one for sure.